atc/engine/engine.go

```diff
@@ -240,7 +240,7 @@ func (b *engineBuild) Run(ctx context.Context) {
 	case err = <-done:
 		logger.Debug("engine-build-done")
 		if err != nil {
-			if _, ok := err.(exec.Retriable); ok {
+			if _, ok := err.(exec.Retryable); ok {
```

Comment:
Not really a big deal, but I was curious about this. According to https://english.stackexchange.com/a/305433, there isn't a "correct" spelling, and both forms are pretty much equal in terms of usage.

Reply:
Ide was complaining about this, so I just changed according to suggestion, I can revert back